importlab/graph.py

```diff
@@ -254,11 +252,14 @@ def get_file_deps(self, filename):
         r = resolve.Resolver(self.path, filename)
         resolved = []
         unresolved = []
+        provenance = {}
         for imp in parsepy.get_imports(filename, self.env.python_version):
             try:
                 f = r.resolve_import(imp)
-                if not f.endswith('.so'):
-                    resolved.append(os.path.abspath(f))
+                if not f.is_extension():
+                    full_path = os.path.abspath(f.path)
```

Comment:
Ran into a bit of an odd issue when playing around with pytype-all and this branch of importlab. The symptom is the following buggy behavior:
```
$ cat foo.py 
import collections
$ pytype-all foo.py
CRITICAL Cannot parse input files:
[Errno 2] No such file or directory: '/usr/local/google/home/rechen/collections.py'
```
What I think is happening is the following:
1. pytype-all adds a PytdFileSystem (`pytd_fs`) with its root at pytype/pytd/stdlib/2and3/.
2. Resolver.resolve_import calls `Resolver._find_file(pytd_fs, 'collections')` and finds pytype/pytd/stdlib/2and3/collections.pytd, so it resolves the file to `Local('collections.py', 'collections', pytd_fs)`.
3. This line converts 'collections.py' to '{cwd}/collections.py'.

Reply:
fixed